### ISSUE TITLE
Parser Error with Umlauts and UTF8 characters

### ISSUE THREAD

**Issue by Sebobo** [cvette/neos-fusion-parser#3]
The following example causes a parser error:

`Fusion parsing error: Unexpected character "'" on line 7 position 26.`

If I remove the `Ü` it works.

```
prototype(Vendor.Package:Component.Test) < prototype(Neos.Fusion:Component) {

    @styleguide {
        description = 'Ü'
    }

    tariffbox_text = 'asd'

    renderer = afx`
        <div>
        </div>
    `
}
```

Or 

```
prototype(Foo.Bar:Test) < prototype(Neos.Fusion:Component) {

    @styleguide {
        props {
            unit = '€/Monat'
            popoverText = '<p>Lorem ipsum dolor sit amet, consetetur sadipscing elitr, </p>
                <p>takimata sanctus est Lorem ipsum dolor sit amet.</p>'
        }
    }
}
```

**Comment by cvette** [comment]
Are you using the latest version? This  should have fixed any utf8 problems: 8758adbed9a7642c1c45e2001e

**Comment by Sebobo** [comment]
Yes I updated before running my tests and I have that version.
